shell/browser/native_window_mac.mm

```diff
@@ -1073,6 +1073,10 @@ void ViewDidMoveToSuperview(NSView* self, SEL _cmd) {
       setSharingType:enable ? NSWindowSharingNone : NSWindowSharingReadOnly];
 }
 
+void NativeWindowMac::SetFocusable(bool focusable) {
+  [window_ setDisableKeyOrMainWindow:!focusable];
```

Comment:
Looking at the Windows implementation, maybe we should also immediately remove focus from the window right after calling `w.setFocusable(false)`?

See the `Focus(false)` call below:
https://github.com/electron/electron/blob/1d8e16bc6ecb5a2700e75e8466f17da9e3391628/shell/browser/native_window_views.cc#L918-L925